src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -59,11 +63,10 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             {
                 type: CONST.ONFIDO.TYPE.DOCUMENT,
                 options: {
-                    useLiveDocumentCapture: true,
                     forceCrossDevice: true,
                     hideCountrySelection: true,
-                    country: 'USA',
```

Comment:
This param wasn't found by TS

Reply:
ok, but we should make sure isn't being used in Onfido SDK, because it can be an issue with TS typings but not the code itself.

Reply:
Since I don't have access to the source code it's hard to say if it's used. But [based on the docs](https://github.com/onfido/onfido-sdk-ui/blob/master/README.md#country-and-document-type-selection---sdk-integration-code) you can pass country to specific document, which is done [here](https://github.com/VickyStash/Expensify/blob/ts-migration/onfido-component/src/components/Onfido/BaseOnfidoWeb.tsx#L71).